tests/src/test/java/org/eclipse/hono/tests/IntegrationTestSupport.java

```diff
@@ -481,6 +490,8 @@
      */
     public AmqpApplicationClientFactory amqpApplicationClient;
 
+    public KafkaApplicationClientFactory kafkaApplicationClientFactory;
```

Comment:
is there also a way to retrieve a factory?

Reply:
I'll remove this. This can be added later on when test code is adapted to Kafka.